There's this typescript error:
```
crates/next-custom-transforms/tests/fixture/server-actions/server-graph/72/input.ts(3,27):
error TS2307: Cannot find module './types' or its corresponding type declarations.
```

--- a/crates/next-custom-transforms/src/transforms/server_actions.rs
+++ b/crates/next-custom-transforms/src/transforms/server_actions.rs
@@ -1819,7 +1819,7 @@ impl<C: Comments> VisitMut for ServerActions<C> {
                         }
                     }
                     ModuleItem::ModuleDecl(ModuleDecl::ExportNamed(named_export))
-                        if named_export.src.is_none() =>
+                        if named_export.src.is_none() && !named_export.type_only =>
                     {
                         for spec in &named_export.specifiers {
                             match spec {

--- a/crates/next-custom-transforms/tests/fixture/server-actions/server-graph/72/input.ts
+++ b/crates/next-custom-transforms/tests/fixture/server-actions/server-graph/72/input.ts
@@ -1,5 +1,6 @@
 'use server'
 
+// @ts-ignore -- that file does not exist
 export type { Item } from './types'
 export type { Foo }
 

--- a/crates/next-custom-transforms/tests/fixture/server-actions/server-graph/72/input.ts
+++ b/crates/next-custom-transforms/tests/fixture/server-actions/server-graph/72/input.ts
@@ -1,0 +1,10 @@
+'use server'
+
+export type { Item } from './types'
+export type { Foo }
+
+type Foo = string
+
+export async function actionA(): Promise<string> {
+  return 'hello from actionA'
+}

--- a/crates/next-custom-transforms/tests/fixture/server-actions/server-graph/72/output.ts
+++ b/crates/next-custom-transforms/tests/fixture/server-actions/server-graph/72/output.ts
@@ -1,0 +1,12 @@
+/* __next_internal_action_entry_do_not_use__ {"0095ef8ede0a8a4c822fcbdb018cb264731cb15281":{"name":"actionA"}} */ import { registerServerReference } from "private-next-rsc-server-reference";
+export type { Item } from './types';
+export type { Foo };
+type Foo = string;
+export async function actionA(): Promise<string> {
+    return 'hello from actionA';
+}
+import { ensureServerEntryExports } from "private-next-rsc-action-validate";
+ensureServerEntryExports([
+    actionA
+]);
+registerServerReference(actionA, "0095ef8ede0a8a4c822fcbdb018cb264731cb15281", null);

--- a/crates/next-custom-transforms/tests/fixture/server-actions/server-graph/72/output.ts
+++ b/crates/next-custom-transforms/tests/fixture/server-actions/server-graph/72/output.ts
@@ -1,4 +1,5 @@
 /* __next_internal_action_entry_do_not_use__ {"0095ef8ede0a8a4c822fcbdb018cb264731cb15281":{"name":"actionA"}} */ import { registerServerReference } from "private-next-rsc-server-reference";
+// @ts-ignore -- that file does not exist
 export type { Item } from './types';
 export type { Foo };
 type Foo = string;

